cmd/buildah/push.go

```diff
@@ -29,6 +31,10 @@ var (
 			Name:  "disable-compression, D",
 			Usage: "don't compress layers",
 		},
+		cli.StringFlag{
+			Name:  "format, f",
+			Usage: "`MANIFEST TYPE` (oci, v2s1, or v2s2) to use when saving image to directory using the 'dir:' transport (default is manifest type of source)",
```

Comment:
I'd make "MANIFEST TYPE" all lower to match the other commands.  Unless @nalind has objections, I'd shorten this to:
"manifest type (oci, v2s1, or v2s2) to use when saving image"

They can dig for details in the man page if that doesn't jog their memory.

Reply:
fixed